lib/mix/lib/mix/task.ex

```diff
@@ -193,7 +193,10 @@ defmodule Mix.Task do
   end
 
   def alias?(task) when is_atom(task) do
-    Mix.Project.config[:aliases][task]
```

Comment:
What about `!!Mix.Project.config[:aliases][task]`?

Can you also please change the tests to check for `== true` and `== false` accordingly?

Thank you!